controller/search.go

```diff
@@ -286,6 +293,28 @@ func (c *SearchController) Spaces(ctx *app.SpacesSearchContext) error {
 		q = "" // Allow empty query if * specified
 	}
 
+	var sort string
+	if ctx.Sort == nil {
```

Comment:
Please use a `default` in the switch of the `else` branch instead.

Reply:
Then have something like this please:

```go
s := ""
if ctx.Sort != nil {
  s = *ctx.Sort
}
switch s {
 //
}
```

It keeps the switch logic separated from figuring out what the parameter is.